Tasks/Denkevich/Task4/Task4.py

```diff
@@ -0,0 +1,57 @@
+tt = open('text.txt', 'r')
+t = tt.read()
+tt.close()
```

Comment:
OK, I promised not to take any actions against reading the whole file, but not using the **with/as**  operators is outrageous, -1 point for that mischief